src/asmcnc/skavaUI/screen_job_recovery.py

```diff
@@ -398,7 +398,7 @@ def go_xy(self):
         self.m.s.write_command('G90 G0 X%s Y%s' % (self.pos_x, self.pos_y))
 
     def back_to_home(self):
-        self.jd.reset_recovery()
+        self.jd.reset_values()
         self.sm.current = 'home'
```

Comment:
does reset_recovery function still get used anywhere? does it need clearing from job data if not?

Reply:
Yep, it's used when writing to the recovery file